Packs/HelloWorld/Scripts/HelloWorldScript/README.md

```diff
@@ -0,0 +1,39 @@
+Hello World Script
+## Script Data
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Tags | helloworld |
+| Demisto Version | 0.0.0 |
+
+## Inputs
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| name | Hello command \- prints hello to anyone. |
+
+## Outputs
+---
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| HelloWorld.hello | Should be Hello \*\*something\*\* here. | String |
+
+
+## Script Example
+```!HelloWorldScript hello=DBot```
+
+## Context Example
+```
+{
+    "HelloWorld": {
+        "hello": "Hello None"
+    }
+}
+```
+
+## Human Readable Output
+## Hello None
```

Comment:
I think there is a bug here. You passed Dbot and got None

Reply:
Fixed